tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -310,9 +315,9 @@ public void testPrometheusFormatNoBadChars() throws Exception {
                 continue;
             }
             String[] tmp = line.split(" ");
-            assert tmp.length == 2;
-            assert !tmp[0].matches("[-.]") : "Line has illegal chars " + line;
-            assert !tmp[0].matches("__") : "Found __ in " + line;
+            assertEquals(tmp.length, 2);
+            assertTrue("Line has illegal chars " + line, !tmp[0].matches("[-.]"));
```

Comment:
Small nitpick - we could use assertFalse here and reverse the logic in the 2nd argument